src/popup/popup.tsx

```diff
@@ -45,20 +43,78 @@ Promise.all([
     );
   });
 
-waitUntilStoreReady(store)
+waitUntilFirstStoreUpdate(store)
```

Comment:
Nice concept of extending

 _module (DOM) loading  -> component first render_ 

to 

 _module (DOM) loading -> store ready -> store init action -> component first render._ 

Clear